README.md

```diff
@@ -136,6 +138,19 @@ To run tests, fork & clone the repository then run `npm install && npm test`.
 
 To run end-to-end tests run `npm run e2e`. This works by fully emulating a run inside the `/test/project` directory and executing `/test/system/run.js` with the `/test/system/tests.json` definitions file.
 
+## Why Devlab Over Docker Compose?
+
+First off, we like [Docker Compose](https://docs.docker.com/compose/), and definitely think it's a powerful tool. However, Devlab was built because Compose is more about long-running, containerized environment and what we set out to build was a way to run ephemeral, limited-lifespan tasks without having to manage cleanup between each run.
+
+Compose takes the approach of spinning up containers that run, almost like a virtual machine, while you need them. Devlab looks at things from a point of view of abstracting `docker run` command chains to create a single-run instance only for that task, then shutting down and doing cleanup so each run is clean and running off a consistent base.
+
+Some more comparisons:
+
+* With Devlab you don't need a Dockerfile for local development, thus you can include one in the root of your project purely for deploying to production, or without a Dockerfile at all.
+* Devlab doesn't build docker images, ever. It uses the images you specify for both the primary container and any services.

```

Comment:
To answer the obvious question this brings up, I'd go ahead and tell folks something to the effect of

>When you install local dependencies in your project folder, run a build, execute your coverage tool, or write any local files, that just happens on your hard disk, not locked away in some container. They'll be available to every other task you run.